source/redirects.py

```diff
@@ -0,0 +1,11 @@
+redirects = {
+     "payment_solutions/recharge": "../../custom_integration/recharge/",
+     "payment_solutions":"../custom_integration/recharge/",
+     "advanced_features/product_sku": "../../advanced_features/product_items/",
+     "advanced_features/traffic_source": "../../integration/custom/integration_components/",
+     "campaigns/views/affiliate_members_new": "../../../campaigns/views/offers_show/",
+     "api_v2/refunds": "../../api_v2/intro/",
+     "api_v2/coupons": "../../api_v2/intro/",
+     "api_v2/visitors": "../../api_v2/intro/",
+     "api_v2/campaigns": "../../api_v2/intro/",
```

Comment:
Let’s have this alphabetically sorted.
```suggestion
     "advanced_features/product_sku": "../../advanced_features/product_items/",
     "advanced_features/traffic_source": "../../integration/custom/integration_components/",
     "api_v2/campaigns": "../../api_v2/intro/",
     "api_v2/coupons": "../../api_v2/intro/",
     "api_v2/refunds": "../../api_v2/intro/",
     "api_v2/visitors": "../../api_v2/intro/",
     "campaigns/views/affiliate_members_new": "../../../campaigns/views/offers_show/",
     "payment_solutions":"../custom_integration/recharge/",
     "payment_solutions/recharge": "../../custom_integration/recharge/",
```